Basic Comcepts/1/date_format.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+date = Time.new
+values = date.to_a
+
+puts 'Different formats are: \n'
```

Comment:
@hmkashif I need output in following manners only
Current Date and Time: 9-07-2019 06:04 AM
Current Date and Time: 9-07-2019 06:04 (in 24 hours format)
Current Date and Time: Tuesday 9 July 2019 06:04 AM

Reply:
Updated!